ctapipe/io/simteleventsource.py

```diff
@@ -340,6 +340,12 @@ def is_compatible(file_path):
 
     @property
     def subarray(self):
+        if self.allowed_tels:
+            return self._subarray_info.select_subarray(
+                self._subarray_info.name
+                + "_subset".join(f"_{t}" for t in self.allowed_tels),
```

Comment:
This can get very long. Maybe better use a notation like `1-4,60-80` like implemented in here: https://github.com/cta-observatory/ctapipe/pull/1557

Reply:
Also didnt change the name in the dl1eventsource